tests/test_base_templates.py

```diff
@@ -559,9 +557,7 @@ def test_ctas_empty_query_creation(expected, schema, table, cols, types):
             ["String", "Int"],
         ),
         (
-            """CREATE TABLE IF NOT EXISTS local_table AS SELECT
-    a,
-    b
+            """CREATE TABLE IF NOT EXISTS "local_table" AS SELECT "a", "b"
```

Comment:
So what is the risk you are guarding against in this PR? Like, that a table name wouldn't have a prefix and also be a reserved word? Cause I assume no table with a study prefix could be a problem. Should we instead/additionally enforce that there is a prefix (look for a dunder)?

Reply:
Specifically, the LOINC groups dataset has a file just named 'Group', and columns named 'group'. I think you're right that, if a dunder was present, this would largely not be an issue. But since this is one of these static dataset situations that you want to use across multiple studies/datasources, I'm inclined to just live with the quoting here?

Reply:
Ah yeah and the "dundering" there is done via a different namespace. Sure - quoting is fine, was just curious if we should go even further.